packages/lib/src/vis/shared/VisCanvas.tsx

```diff
@@ -54,34 +54,31 @@ function VisCanvas(props: PropsWithChildren<Props>) {
 
   return (
     <div
-      className={styles.canvasWrapper}
+      className={styles.visCanvas}
       style={{
-        paddingBottom: axisOffsets.bottom,
-        paddingLeft: axisOffsets.left,
-        paddingTop: axisOffsets.top,
-        paddingRight: axisOffsets.right,
+        gridTemplateColumns: `${axisOffsets.left}px minmax(0, 1fr) ${axisOffsets.right}px`,
+        gridTemplateRows: `${axisOffsets.top}px minmax(0, 1fr) ${axisOffsets.bottom}px`,
       }}
     >
-      <R3FCanvas className={styles.r3fRoot} orthographic>
-        <VisCanvasProvider
-          visRatio={visRatio}
-          abscissaConfig={abscissaConfig}
-          ordinateConfig={ordinateConfig}
-          svgOverlay={svgOverlay}
-          floatingToolbar={floatingToolbar}
-        >
-          <AxisSystem
-            axisOffsets={axisOffsets}
-            title={title}
-            showAxes={showAxes}
-          />
-          <InteractionsProvider>{children}</InteractionsProvider>
-          <ViewportCenterer />
-          <RatioEnforcer />
-          {raycasterThreshold !== undefined && (
-            <ThresholdAdjuster value={raycasterThreshold} />
-          )}
-        </VisCanvasProvider>
+      {showAxes && title && <p className={styles.title}>{title}</p>}
+
+      <div className={styles.canvasWrapper}>
```

Comment:
This is the new wrapper. Couldn't think of a better name for now. With the CSS Grid on `visCanvas`, things might change a bit more and this wrapper might in fact be removed.

Reply:
`r3fRootContainer` ?

Reply:
Sure. It was a wild suggestion

Reply:
#1479